ptbcontrib/username_to_chat_api/README.md

```diff
@@ -38,10 +40,13 @@ logging.basicConfig(
 class CustomContext(CallbackContext):
     def __init__(self, application: Application):
         super().__init__(application=application)
-        self._wrapper: UsernameToChatAPI = UsernameToChatAPI("https://localhost:1234/", "RationalGymsGripOverseas", application.bot)
+        if "wrapper" not in application.bot_data:
+            wrapper: UsernameToChatAPI = UsernameToChatAPI("https://localhost:1234/", 
+                                                           "RationalGymsGripOverseas", application.bot)
+            application.bot_data["wrapper"] = wrapper
```

Comment:
uhm, stupid question again: why do this here and not just directly in the main with `application.bot_data["wapper"] = UsernameToChatApi(…)`? The main advantage of using a custom context class would be that you e.g. have it directly accessible via `context.wrapper` - for that I would do something like 

```python
class  CustomContext(CallbackContext[insert type vars here]):
    @property
    def wrapper(self) -> UsernameToChatAPI:
        return self.bot_data["wrapper"]
```